src/components/VideoListing/VideoListing.js

```diff
@@ -44,7 +44,7 @@ export const VideoListing = () => {
                 <div className="info-container">
                   <img className="avatar" src={avatar} alt={channelName} />
                   <div className="video-info">
-                    <h3 className="video-title">{trimExtraChars(title)}</h3>
+                    <h3 className="video-title">{trimExtraChars(title, 50)}</h3>
                     <span>{channelName}</span>
```

Comment:
I liked how you createda trim function. Nice work

Reply:
Thank you